test/fixtures/access-control/common/models/account2.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "accountWithReplaceOnPUTfalse",
+  "plural": "accounts2",
```

Comment:
Please use a more descriptive URL path than `accounts2` (and `accounts` for the other model). How about `accounts-replacing` and `accounts-updating`?